src/main/java/com/google/devtools/build/lib/remote/merkletree/MerkleTree.java

```diff
@@ -68,21 +69,37 @@ public ByteString getBytes() {
   private final Map<Digest, Directory> digestDirectoryMap;
   private final Map<Digest, PathOrBytes> digestFileMap;
   private final Digest rootDigest;
+  private final long inputFiles;
+  private final long inputBytes;
 
   private MerkleTree(
       Map<Digest, Directory> digestDirectoryMap,
       Map<Digest, PathOrBytes> digestFileMap,
-      Digest rootDigest) {
+      Digest rootDigest,
+      long inputFiles,
+      long inputBytes) {
     this.digestDirectoryMap = digestDirectoryMap;
     this.digestFileMap = digestFileMap;
     this.rootDigest = rootDigest;
+    this.inputFiles = inputFiles;
+    this.inputBytes = inputBytes;
   }
 
   /** Returns the digest of the merkle tree's root. */
   public Digest getRootDigest() {
     return rootDigest;
   }
 
+  /** Returns the number of files represented by this merkle tree */
+  public long getInputFiles() {
+    return inputFiles;
+  }
+
+  /** Returns the number of bytes represented by this merkle tree, including directories */
```

Comment:
Returns the sum of file sizes plus protobuf sizes used to represent this merkle tree?

Reply:
Yes. I assume that's what you want me to change the comment to be?